config/eslint.js

```diff
@@ -46,7 +57,32 @@ module.exports = {
     '@typescript-eslint/no-redeclare': ['error'],
     '@typescript-eslint/restrict-plus-operands': 'off',
     'import/no-default-export' : ['error'],
-    '@typescript-eslint/strict-boolean-expressions': ['error']
+    '@typescript-eslint/strict-boolean-expressions': ['error'],
+    // 'simple-import-sort/imports': 'error',
```

Comment:
Instead of commenting out, I'd suggest we just remove it and add it to our linting issue/discussion.